alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -20,9 +20,10 @@ library
   if impl(ghc < 8.10)
     buildable: False
   exposed-modules:
+    Cardano.Ledger.DescribeEras
```

Comment:
This isn't to do with Alonzo. However, there is a new test package `cardano-ledger-test`, which sits atop all eras and is intended to be the home for new tests that use all eras - this would be a perfect fit for that package.